### PR TITLE
Bug fix for #463: Handle Date and Decimal types

### PR DESCRIPTION
**Pull request by quanghgx**
* Handle Date and Decimal types
* Data for unit-test is from @cristichircu in #463

### CODE REVIEW

File: src/main/scala/com/crealytics/spark/v2/excel/ExcelParser.scala [hunk 1]
```diff
@@ -247,7 +254,7 @@ class ExcelParser(dataSchema: StructType, requiredSchema: StructType, val option
               datum.getCellType match {
                 case CellType.NUMERIC | CellType.FORMULA => {
                   /* Excel to spark precision */
-                  datum.getNumericCellValue * 1000
```
Comment (nightscape) [inline src/main/scala/com/crealytics/spark/v2/excel/ExcelParser.scala:250]: Why is this not needed anymore? Does Spark do something differently if it gets a Long value?

Comment (quanghgx) [reply src/main/scala/com/crealytics/spark/v2/excel/ExcelParser.scala:250]: Hi Martin,

It was incorrect before. This code section gets called when a cell is not formatted as Date or Time, however, users pass a custom schema and want to load that cell value as Timestamp, so there is no binding logic to multiply it to 1000.

We haven't seen a case like this in real life yet. And only when handling DateType, I recognized this issue.

**Comment by nightscape** [comment]
LGTM! Merge at your convenience 👍

**Comment by cristichircu** [comment]
@quanghgx are you waiting for something from my side or are you busy with somehting else? Just trying to make sure I didn't miss a request or something :) Thank you!

**Comment by nightscape** [comment]
@quanghgx @cristichircu I just went ahead and merged this. It's got tests so we should be good 👍

**Comment by quanghgx** [comment]
Thank you Martin.
Hi @cristichircu , it's just that I am lacking the time to put it forward.

**Pull request merged**